pkg/common/helpers/parser.go

```diff
@@ -0,0 +1,15 @@
+package helpers
+
+import "strings"
+
+func GetAwsAccountIdAndClusterName(clusterArn string) (string, string) {
```

Comment:
some comments on both of this func will be helpful.

Reply:
Checked that comments has been added.